services/src/pro/handlers/drone_mapping.rs

```diff
@@ -422,6 +423,11 @@ mod tests {
         let ctx = ProInMemoryContext::test_default();
         let session = create_session_helper(&ctx).await;
 
+        ctx.user_db_ref()
+            .update_quota_available_by_user(&session.user.id, 9999)
+            .await
+            .unwrap();
```

Comment:
Should we really put this into this test? It is not really part of what we want to test.

Reply:
by default, in all tests, the available quota is zero. We could change it and have by default 9999 and only set it to zero in the tests about quotas

Reply:
That sounds about right. I don't know about the 9999 part. Is this the maximum quote we can think of? :smile:

Reply:
it's big enough for the tests and short enough to be read

Reply:
If it should never fail, maybe int max?

Reply:
that's a bulky number, but if you insist I can change it. which int would you like? 32 or 64 bit? signed or unsigned?